src/main/java/org/sunbird/common/util/Constants.java

```diff
@@ -1316,9 +1316,10 @@ public class Constants {
 	public static final String ENABLED = "enabled";
 
 	public static final String INVALID_GROUP_MESSAGE = "Invalid Group : Group can be only among one of these ";
-  public static final String SEND_FOR_APPROVAL = "SEND_FOR_APPROVAL";
+    public static final String SEND_FOR_APPROVAL = "SEND_FOR_APPROVAL";
 	public static final String ERR_MSG_APPROVAL_REQUEST = "Error while rejecting approval requests";
 	public static final String FAILED_MSG_APPROVAL_REQUEST = "Failed to reject approval requests for userId: {}";
+	public static final String PROFILE_APPROVAL_AUTO_REJECT_MESSAGE ="You are no longer recognized as a user by your current organization. Your access will expire in 48 hours. Reach out to your MDO or submit a \"Transfer Request\".";
```

Comment:
why \"Transfer Request\". ?
it should be string

Reply:
I got below text from Prakhar for comment on auto rejection of profile approval if admin marks a user NOT-MY-USER: 
 
You are no longer recognized as a user by your current organization. Your access will expire in 48 hours. Reach out to your MDO or submit a "Transfer Request".